middleware/logger/logger.go

```diff
@@ -85,9 +85,6 @@ func New(config ...Config) fiber.Handler {
 			return c.Next()
 		}
 
-		// Alias colors
-		colors := c.App().Config().ColorScheme
```

Comment:
Why was this removed?

Reply:
Because this variable was not used anymore

Reply:
i checked it again -> its fine